manager/src/main/java/com/redhat/service/smartevents/manager/connectors/ConnectorsServiceImpl.java

```diff
@@ -48,27 +48,39 @@ public void createConnectorEntity(Processor processor) {
         if (processor.getType() == ProcessorType.SOURCE) {
             createConnectorEntity(processor, processor.getDefinition().getRequestedSource());
         } else {
-            createConnectorEntity(processor, processor.getDefinition().getRequestedAction());
+            if (!(processor.getDefinition().isErrorHandler())) {
```

Comment:
This could be wrong; but I didn't get a chance to test with a Managed Connector Error Handler; e.g. `Slack`.

Managed Connector integration worked fine when the User had to define a secondary `Processor` manually (that only supported a hard-coded Webhook `Action`) but following the changes to move it to the `BridgeAPI` I suspect this check is now obsolete and can be removed; since a Error Handler can now use *any* `Action`. 

I don't have time to fix.